scripts/generate_go.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# protoc needs this to build go code
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+export PATH="$PATH:$(go env GOPATH)/bin"
+
+# make the generated folder if it doesnt already exist
+mkdir -p generated
+
+protoc \
+--proto_path=messages \
+--go_out=generated \
```

Comment:
confirming this will generate messages into `generated/go`? I've been putting the c stuff into `generated/c`

Reply:
nah this puts it directly into generated. i can put it in generated/go for now but it really hate this level of nesting so I may change it up later on

Reply:
i think language-level nesting is ok - CMake right now has to just include the entire `generated/c/` path so that it can find the messages